### PR TITLE
docs: remove unsupported Filter code sample

### PR DESCRIPTION
**Pull request by kenjis**
See https://github.com/codeigniter4/shield/discussions/600#discussioncomment-4691901

- [$filters](https://codeigniter.com/user_guide/incoming/filters.html#filters) cannot set filer arguments.

### CODE REVIEW

File: docs/authorization.md [hunk 1]
```diff
@@ -130,28 +129,34 @@ if (! $user->hasPermission('users.create')) {
...
-
-The filters can also be used on a route or route group level:
+You can set the filters within **app/Config/Routes.php**:
 
 ```php
 $routes->group('admin', ['filter' => 'group:admin,superadmin'], static function ($routes) {
-    $routes->resource('users');
+    $routes->group(
+        '',
+        ['filter' => ['group:admin,superadmin', 'permission:users.manage']],
```
Comment (datamweb) [inline docs/authorization.md:147]:
```suggestion
        ['filter' => ['group:admin,superadmin', 'permission:users.manage-admins']],
```

**Review by datamweb — APPROVED** [review]
Thanks @kenjis and sorry for the delay.

File: docs/authorization.md (hunk 1 above)
Comment (kenjis) [reply docs/authorization.md:147]: `permission:users.manage` is not listed in https://github.com/codeigniter4/shield/blob/develop/docs/authorization.md#defining-available-permissions
But in this case,  I feel `permission:users.manage-admins` is a bit weird.
The URI seems the page to manage users, not admin-users.

**Comment by kenjis** [comment]
@datamweb No problem. Thank you for the review.

**Pull request merged**